### ISSUE TITLE
Slashes should be normalized in paths

### ISSUE THREAD

**Issue by martinml** [revel/revel#260]
_**tl;dr**: Windows uses `\` as a path separator and that messes up `BasePath` and possibly others_

While reading docs and source I kept reading that Revel included a `PanicFilter` that would create nice error pages in case of `panic()`, but it wasn't working for me: in dev or in prod mode, I was seing always the raw stacktrace being dumped to my browser.

Perusing the source I arrived to `findRelevantStackFrame()` in `errors.go`, which I guess tries to find out if the panic was inside the application by finding `BasePath` in the stack trace. The problem is that if you use Windows, it never finds it, because the slashes are wrong:

 - stack = `(...) C:/gocode/src/myapp/app/controllers/whatever.go (...)`
 - BasePath = `C:\gocode\src/myapp`

Note the mixing of different slashes. Windows accepts both so everything works, but if you search `BasePath` in `stack`, it won't find it.

A solution might be replacing in `SourcePath` all the backslashes (`\`) with forward slashes (`/`) _but only if we are in Windows_, but that seems a bit hacky. What would be a better solution?

**Comment by jeevatkm** [comment]
#1028 PR is merged to `develop` branch for handling path separator using `filepath.ToSlash` in `error.go`. So I'm closing this one.
